infobip-typescript-generator-extension-api/src/main/java/com/infobip/typescript/NoParametersDecorator.java

```diff
@@ -0,0 +1,13 @@
+package com.infobip.typescript;
+
+import java.lang.annotation.Annotation;
+import java.util.Collections;
+import java.util.List;
+
+public class NoParametersDecorator implements DecoratorParameterListExtractor{
```

Comment:
tu nisi parametrizirao ovaj DecoratorParameterListExtractor, znam da je nebitno al IDEA se buni :) mozes staviti samo <Annotation> prilicno sam siguran

Reply:
e ali nemre se ni parametrizirat jer je to kao dekorator koji nema parametre pa se ni ne zna na koji je točno anotaciju vezan tako da on stvarno i je neparamtriziran :( tako da bi ja to za sad ostavio jer je to u principu da podržimo i custom anotacije ali koje nemaju paramtre